hpx/components/security/certificate.hpp

```diff
@@ -119,7 +120,7 @@ namespace hpx { namespace components { namespace security
 #endif
 }}}
 
-namespace boost { namespace serialization
+namespace hpx { namespace traits {
 {
```

Comment:
The security code in HPX is not actively used and not functional. I think we should remove it soon.